vnext/local-cli/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "@rnw-scripts/ts-config",
+  "compilerOptions": {
+    "outDir": "lib-commonjs",
```

Comment:
I should explicitly specify CommonJS in the root tsconfig instead of having just-task specify the same for everything.